frontend/src/metabase/visualizations/components/ChartSettingsWidget.styled.tsx

```diff
@@ -72,6 +73,14 @@ export const Title = styled.label<VariantProp>`
     css`
       font-weight: 700;
     `}
+
+  ${props =>
```

Comment:
Should we maybe apply this style for all places instead of introducing a new variant?

Reply:
Right. I changed it so that we only have the 1 variant in the `Title` component